README.md

```diff
@@ -1,49 +1,46 @@
-# Welcome to [Astro](https://astro.build)
+# CachyOS Website
 
-[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/withastro/astro/tree/latest/examples/basics)
-[![Open with CodeSandbox](https://assets.codesandbox.io/github/button-edit-lime.svg)](https://codesandbox.io/s/github/withastro/astro/tree/latest/examples/basics)
+Static site for CachyOS, built with Astro. Origin: `https://cachyos.org/`.
 
-> 🧑‍🚀 **Seasoned astronaut?** Delete this file. Have fun!
+## About CachyOS
```

Comment:
don't think that's required to be here. overwise can be linked to website or wiki page for that